examples/web/index.html

```diff
@@ -155,6 +155,10 @@ <h2 class="typography-heading-medium-text mt-1100 mb-800">TextField</h2>
               <label for="textfield1" class="TextField__label TextField__label--required">Label</label>
               <input type="text" id="textfield1" class="TextField__input" placeholder="Placeholder" />
             </div>
+            <div class="TextField">
```

Comment:
@crishpeen Please display all validation states like it was in PasswordField. Here in TextField is only an error state displayed. Thanks.

Reply:
Thanks, added.